metadata/junit/junit.go

```diff
@@ -172,7 +175,7 @@ func truncatePointer(str *string, max int) {
 
 // Truncate ensures that strings do not exceed the specified length.
 func (r Result) Truncate(max int) {
-	for _, s := range []*string{r.Failure, r.Skipped, r.Error, r.Output} {
+	for _, s := range []*string{r.Error, r.Failure, r.Skipped, r.Output} {
```

Comment:
Think this still needs to account for r.Errored too?

Reply:
Made a mistake, fixed.